microprofile/server/src/main/java/io/helidon/microprofile/server/JaxRsService.java

```diff
@@ -97,6 +98,12 @@ static JaxRsService create(ResourceConfig resourceConfig, InjectionManager injec
         resourceConfig.property(PROVIDER_DEFAULT_DISABLE, "ALL");
         resourceConfig.property(WADL_FEATURE_DISABLE, "true");
 
+        // TODO - temporary until MP OpenAPI TCK bug fix released. https://github.com/eclipse/microprofile-open-api/issues/557
+        if (System.getProperties().containsKey("io.helidon." + ServerProperties.RESOURCE_VALIDATION_IGNORE_ERRORS)) {
```

Comment:
Strictly speaking probably `System.getProperties().stringPropertyNames().contains(...)`

Reply:
Would this code block behave differently or more safely with the change? 

I think that if, somehow, the property had been defined as a value with a type other than `String` (not possible using `setProperty` but possible using `put` from the `Map` interface), then with the code as written the subsequent `Boolean.getBoolean` using that key would return `false`, the default setting for `RESOURCE_VALIDATION_IGNORE_ERRORS`.

Using the suggested change, the returned set of keys would exclude the one of interest because the value is a non-`String`, so the code would not set the `resourceConfig` property `RESOURCE_VALIDATION_IGNORE_ERRORS` to anything, so its default `false` would reign.

Reply:
Good point. I withdraw the observation your honor